src/app/order/table.tsx

```
@@ -1,117 +1,137 @@
 import dayjs from 'dayjs';
-import {KeyboardArrowDown, KeyboardArrowUp} from '@mui/icons-material';
+import { KeyboardArrowDown, KeyboardArrowUp, RefreshRounded, SettingsBackupRestore} from '@mui/icons-material';
+import ImageIcon from '@mui/icons-material/Image';
 import { usePathname, useRouter, useSearchParams } from 'next/navigation';
 import Link from 'next/link';
 import {SelectState} from "../components/selecState"
 import {colors} from "../utils/colors"
-import { PaginationTable } from "../components/pagination/pagination";
+import {PaginationTable} from "../components/pagination/pagination";
 import {UpdateDatePedido, UpdateStatePedido, sendNovedad, resetPedido} from "../store/fetch-pedido"
 import {Date} from "../components/date"
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)